src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -75,6 +75,8 @@ public class BuildApk : Task
 
 		public string Debug { get; set; }
 
+		public bool UseRuntimeWithDebugSymbols { get; set; }
```

Comment:
There are two `AddNativeLibrary` overloads - one for runtime and another for "user" libraries. The reasoning here is that it's us who follow the ".d.so" naming convention and user would just not strip their libraries instead.